src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../../common/variables';
+
+.bcs-AnnotationActivityMenu-unresolveAnnotation path {
+    fill: $bdl-gray;
```

Comment:
Why is this needed? It looks like all the other icons use `bdl-gray-50` for their paths.

Reply:
All icons for comments are using `bdl-gray`, for annotations trash was different, but now it's the same as for comments. With this change this is how the whole menu looks like:
<img width="187" alt="Screen Shot 2022-11-09 at 22 38 32" src="https://user-images.githubusercontent.com/23643342/200948415-33ba459c-23a8-4972-bb94-d63195bb5549.png">